changelogs/fragments/50615-rpm_key.yml

```diff
@@ -0,0 +1,7 @@
+minor_changes:
+  - >
+    rpm_key - Check for multiple signing keys in a single GPG key file. 
```

Comment:
Could you rephrase this in non-imperative mood? The writing style is inconsistent with the following entry (and likely many other changelog fragments).
This is going to show up in the aggregated change log document on release. That document refers to the changes that have already happened since the previous release, so it's natural to describe things as if they are already merged. Imperative mood is a better fit for Git commit summary lines and TODO entries, but change logs have different style, purpose and audience.